src/cockpit/samples.py

```diff
@@ -121,21 +132,27 @@ def detect_cpu_sensors(self, hwmonid: int, name: str) -> None:
                 if name != 'cpu_thermal':
                     continue
 
-            self.sensors.append(sensor_path)
+            # need full path
+            self.sensors.append(f'{HWMON_PATH}/{hwmon_name}/{sensor}')
 
     def sample(self, samples: Samples) -> None:
         cpu_names = ['coretemp', 'cpu_thermal', 'k8temp', 'k10temp', 'atk0110']
```

Comment:
This probably deserves to be a top-level global constant called `CPU_NAMES`.  Or just open-coded at its single point-of-use.